src/engine/experimental/ePBS.md

```diff
@@ -0,0 +1,115 @@
+# Engine API -- ePBS
+
+Engine API changes introduced in ePBS, based on [Cancun](../cancun.md).
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Structures](#structures)
+  - [InclusionListV1](#inclusionlistv1)
+  - [PayloadStatusVePBS](#payloadstatusvepbs)
+  - [InclusionListStatusV1](#inclusionliststatusv1)
+
+- [Methods](#methods)
+  - [`engine_newPayloadVePBS`](#engine_newpayloadvepbs)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+  - [`engine_getInclusionListV1`](#engine_getinclusionlistv1)
+    - [Request](#request-1)
+    - [Response](#response-1)
+    - [Specification](#specification-1)
+  - [`engine_newInclusionListV1`](#engine_newinclusionlistv1)
+    - [Request](#request-2)
+    - [Response](#response-2)
+    - [Specification](#specification-2)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Structures
+
+### InclusionListV1
+This structure contains a list of transactions that are on the inclusion list. The fields are encoded as follows:
+- `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
+
+### PayloadStatusVePBS
+This structure has the syntax of [`PayloadStatusV1`](../paris.md#payloadstatusv1) and appends a single field: `inclusionList`.
+
+- `status`: `enum` - `"VALID" | "INVALID" | "SYNCING" | "ACCEPTED" | "INVALID_BLOCK_HASH"`
+- `latestValidHash`: `DATA|null`, 32 Bytes - the hash of the most recent *valid* block in the branch defined by payload and its ancestors
+- `validationError`: `String|null` - a message providing additional details on the validation error if the payload is classified as `INVALID` or `INVALID_BLOCK_HASH`.
+- `inclusionList`: [`InclusionListV1`](#inclusionlistv1) // TODO: Should this be `InclusionListV1` or just a simple array of tranasactions? I haven't seen a nested structure in previous specs.
```

Comment:
is there a url that explains the reasoning  of this list in payload status? may be i am missing context

Reply:
No, it’s just an arbitrary status for now to indicate if the said IL is valid or not

Reply:
I think there should be no changes to notifyNewPayload, the engine validation will change in the EIP, but the CL has no use for any return status other than INVALID if the IL has failed to verify